packages/FieldTypes/README.md

```diff
@@ -0,0 +1,76 @@
+<!-- autogenerated don't modify -->
+
+# @paprika/field-types - 0.1.0
+
+## Description
+
+Field Types package is a set of components that help the Table and DataGrid to enhance the cell property to display different value in a ease manner
```

Comment:
Nitpick: adjust grammar slightly 'different values in an easy manner'

Reply:
ohhh there it's an option that you can make the change by yourself in github and propose the change and I can commit it from my side, try it. is the icon named "insert suggestion", use it :D